cmd/extract.go

```diff
@@ -0,0 +1,108 @@
+package cmd
+
+import (
+	"os"
+	"io"
+	"fmt"
+
+	"github.com/sector-f/jh_extract/nvc"
+)
+
+func extractNVC(arcPath string, pathlist []string, outputDirectory string, extractUnknown bool) error {
+	r, err := os.Open(arcPath)
+	if err != nil {
+		return err
+	}
+
+	hashToPath := map[nvc.Hash]string{}
+	for _, p := range pathlist {
+		hash := nvc.String2Hash(p)
+		hashToPath[hash] = p
+	}
+
+	var tocEntries []nvc.TocEntry
+	tocEntries, err = nvc.ReadToc(r)
```

Comment:
Just do `tocEntries, err := nvc.ReadToc(r)` (instead of declaring `tocEntries` separately)

Reply:
:boom: